test/examples/test_kissgp_gp_classification.py

```diff
@@ -28,11 +28,7 @@ def __init__(self):
         self.covar_module = RBFKernel(
             log_lengthscale_prior=SmoothedBoxPrior(exp(-5), exp(6), sigma=0.1, log_transform=True)
         )
-        self.register_parameter(
-            name="log_outputscale",
-            parameter=torch.nn.Parameter(torch.Tensor([0])),
-            prior=SmoothedBoxPrior(exp(-5), exp(6), sigma=0.1, log_transform=True),
```

Comment:
Did you mean to get rid of the prior here?

Reply:
Can we keep the prior and still use `self.my_param = ...` to create a parameter?

Reply:
Well you can, but you'd have to manually call `set_parameter_priors` to set the prior. The `self.my_param = ...` syntax doesn't really allow for a clean way to set a prior I think.

Reply:
Yeah, I changed it here so that we had one unittest that did things that way